docs/src/ComboBox.doc.js

```diff
@@ -0,0 +1,732 @@
+// @flow strict
+import type { Node } from 'react';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="ComboBox"
+    description="ComboBox is the combination of a Textfield and an associated Dropdown to let the user filter a list when selecting. ComboBox allows to type the selection, type part of the selection and narrow the results, and select an option from the list."
```

Comment:
This second sentence feels a bit off. Maybe:

ComboBox is the combination of a Textfield and an associated Dropdown that allows the user to filter a list when selecting an option. ComboBox allows users to type the full option, type part of the option and narrow the results, or select an option from the list.

Reply:
DONE